Automattic/vip-cli
# Block import of multisite into single site and vice versa

## Description

We should fail in the instances that a:

* multisite sql dump file was provided for a single site
* single site sql dump file was provided for a multisite

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Execute an import that would pass -- script continues with import
1. Execute an import that would fail because the SQL dump is multi and the site is not -- scripts exits
1. Execute an import that would fail because the SQL dump is single site and the site is multisite -- script exits

Example commands coming!

---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.3.2)_
